iree/hal/drivers/BUILD

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["-layering_check"],  # buildozer: disable=no-layering-check, allow indirect headers
```

Comment:
comment why this is necessary?

Reply:
Possibly temporary while sorting out the merge. Affects Copybara but not open source. I'm replacing the 
```
"//iree/hal/vulkan/registration",
```
dep with a forwarding dep which doesn't list
```
#include "iree/hal/vulkan/registration/driver_module.h"
```
in its headers (because it forwards...)

"allow indirect headers" captures that.